CHANGELOG.md

```diff
@@ -5,6 +5,11 @@
 * BraintreeCore
   * Allow uppercase characters in default return url scheme
 
+* PayPalNativeCheckout (BETA)
+  * Bump native-checkout version to `0.8.8`
+  * Adding in a requirement for `returnUrl` inside of `PayPalNativeRequest` this url is needed to redirect after authentication
```

Comment:
```suggestion
  * Breaking change
    *`PayPalNativeRequest` requires a `returnUrl` to redirect correctly after authentication
```
Thinking we should maybe call this out as a breaking change

Reply:
+1 on the breaking change callout.